api/tests/acceptance/application/certification-courses/certification-course-controller_test.js

```diff
@@ -885,31 +831,46 @@ describe('Acceptance | API | Certification Course', function () {
         });
 
         it('should have created a V2 certification course', async function () {
+          // given
+          const { options, userId, sessionId } = _createRequestOptions();
+          _createNonExistingCertifCourseSetup({ learningContent, userId, sessionId });
+          await databaseBuilder.commit();
+
           // when
           response = await server.inject(options);
 
           // then
           const certificationCourses = await knex('certification-courses').where({ userId, sessionId });
           expect(certificationCourses).to.have.length(1);
-          expect(certificationCourses[0].version).to.equal(2);
+          expect(certificationCourses[0].version).to.equal(CertificationVersion.V2);
         });
 
         context('when the session is v3', function () {
-          it('should create a v3 certification course', async function () {
+          it('should create a v3 certification course without any challenges', async function () {
```

Comment:
Pour harmoniser avec le test v2
```suggestion
          it('should have created a v3 certification course without any challenges', async function () {
```